geode-docs/managing/security/implementing_ssl.html.md.erb

```diff
@@ -61,15 +61,15 @@ involve a TCP connection, so SSL does not apply.</dd>
 
 Specifying that a component is enabled for SSL applies to the component's server-socket side and its
 client-socket side.  For example, if you enable SSL for locators, then any process that communicates
-with a locator must also have SSL enabled.
+with a locator must also have SSL enabled.  If you provide "" as the value, SSL is turned off for all components.
 
 ## <a id="ssl_configuration_properties" class="no-quick-link"></a>SSL Configuration Properties
 
 You can use <%=vars.product_name%> configuration properties to enable or disable SSL, to identify SSL ciphers and
 protocols, and to provide the location and credentials for key and trust stores.
 
 <dt>**ssl-enabled-components**</dt>
-<dd>List of components for which to enable SSL. Component list can be "all" or a comma-separated list of components.</dd>
+<dd>List of components for which to enable SSL. Component list can be "" (disable SSL), "all" or a comma-separated list of components.</dd>
```

Comment:
Suuuper minor grammatical suggestion: put a comma after "all", so that there are clearly three commma-separated options.